_posts/blog/2018-10-11-4rios-review.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "QUEERING THE DIGITAL ARCHIVE: '4 RIOS' AND GRAPHIC NARRATION"
+modified:
+categories: blog
+excerpt: ...the unique construction of this archive calls into question institutionalized norms regarding what an archive is and how it functions.
+tags: []
+author: "Kaitlin Burns"
+image:
+	feature:
+---
+# **Queering the Method: "4 Ríos" and Archival Narration**
+
+4 Ríos Digital Archive. http://4rios.co. Acccessed 28 Sept. 2018.
+
+The "4 Ríos" digital archive uses artful innovation to paint the indigenous and afro-descendant population of Colombia in the crossfire of armed conflict that has endured over more than forty years. This digital collection inspires the question, "What is an archive" as it reshapes notions on how information must be organized and presented. A truly multi-media experience, "4 Ríos" compiles video, testimony, art, and online sources in the larger framework of an interactive, graphic novel. Visitors to the archive participate in their own education regarding how the lives of Colombian people were spent like currency in the war between paramilitary and military forces, while witnessing information on the massacre at Alto de Naya in April 2001.
+
+Elder Manuel Tobar, an Audiovisual Producer and Social Activist by trade, created and directed the "4 Ríos" information collective in 2013. The description of his extensive team of specialists and their roles reads like movie credits, since the execution of this archive required unique talents such as actors, set designers, and story board creators. The project received the Premio Crea Digital in 2013 in recognition of its narrative value and commitment to social and political responsibility, and was funded through awards and grants from IDARTES (Instituto Distrital de las Artes) and Pontificia Universidad Javeriana in 2012, as well as through the Colombian Ministry of Culture in 2011.
+
+The stated goal of "4 Ríos" is to "propose other forms of approaching memory, to the construction of spaces of exchange and debate, to raise awareness about these facts, to not forgetting, to the rescue of that which, through the passing of time, is believed to be lost" ([Proyecto](http://4rios.co/blog/proyecto/), *my translation*). This is achieved by recounting experiences, facts, and the background and succession of evens in recent history of armed conflict in Colombia - specifically the 2001 massacre on the river Naya.
```

Comment:
I found the first sentence confusing. Consider: "propose other forms of approaching memory, of constructing spaces... , of raising awareness... , of not forgetting, and of rescuing..."

Reply:
thanks! sometimes its hard to get outside of a translation and read it without the original voice!

Reply:
do you think there is any problem with changing the original syntax? The original phrase is a bit convoluted itself...

Reply:
The stated goal of "4 Ríos" is to "propose other forms of approaching memory, to construct spaces of exchange and debate, to raise awareness about these facts, to not forget, to rescue that which, through the passing of time, is believed to be lost" 

Does this work better? if you break it down, it now reads: the goal is to propose, to construct, to raise awareness, etc

Reply:
This works a lot better! Excellent.